src/pages/index.js

```diff
@@ -16,33 +18,65 @@ const profileNameInput = document.getElementById("profile-name-input");
 
 profileEditButton.addEventListener("click", () => {
   const userInfo = profileUserInfo.getUserInfo();
-  profileDescriptionInput.value = userInfo.description;
   profileNameInput.value = userInfo.name;
+  profileDescriptionInput.value = userInfo.description;
   profileModal.open();
 });
 
 addCardButton.addEventListener("click", () => {
   newCardModal.open();
 });
 
+const deleteModal = new PopupConfirm({ popupSelector: "#are-you-sure" });
+
+deleteModal.setEventListeners();
+
 const newCardModal = new PopupWithForm(
   "#profile-card-modal",
   handleCardContent,
-  "#card-add-button"
+  "#card-add-button" // TODO do we need this third argument?
 );
 
 newCardModal.setEventListeners();
+
 const profileName = document.querySelector(".profile__name");
 const profileDescription = document.querySelector(".profile__description");
-const profileUserInfo = new UserInfo({ profileName, profileDescription });
+const profileAvatar = document.querySelector(".profile__image");
+const avatarEditBtn = document.querySelector(".pencil-icon");
+const profileUserInfo = new UserInfo({
+  profileName,
+  profileDescription,
+  profileAvatar,
+});
 
 const profileModal = new PopupWithForm("#profile-edit-modal", (userInput) => {
-  profileUserInfo.setUserInfo(userInput); //used when submitting modal
-  profileModal.close();
+  api
+    .editProfile(
+      JSON.stringify({ name: userInput.name, about: userInput.description })
```

Comment:
`JSON.stringify` should be only inside `Api` because it's the responsibility of that class.